linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/tests/banner_etc_issue_net_disa_dod_default_banner.pass.sh

```diff
@@ -1,29 +1,29 @@
 #!/bin/bash
-#
+# platform = multi_platform_ubuntu
 # profiles = xccdf_org.ssgproject.content_profile_stig
 
 # dod_default banner
-echo "You are accessing a U.S. Government (USG) Information System (IS) that is
-provided for USG-authorized use only. By using this IS (which includes any
+echo "You are accessing a U.S. Government (USG) Information System (IS) that is 
```

Comment:
Was it intentional to have these spaces at the end of the lines?

Reply:
They were not intentional. They were actually inherited from `banner_etc_issue` rule. I will fix there too in order avoid this issue be replicated in the future by copy and paste.